src/renderer/index.ejs

```diff
@@ -6,6 +6,7 @@
   <meta http-equiv="Content-Security-Policy" />
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Feishin</title>
+  <script src="settings.js"></script>
```

Comment:
This should only be loaded for `web` target (it's not useful for desktop Feishin). See https://github.com/jeffvli/feishin/pull/387/files for webpack changes you can do